R/assertions.R

```diff
@@ -1157,7 +1206,11 @@ assert_unit <- function(dataset,
       class = c(class, "assert-admiraldev")
     )
   }
-  if (tolower(units) != tolower(required_unit)) {
+
+  if (!is.null(required_unit) && !(tolower(units) %in% tolower(required_unit))) {
```

Comment:
Out of curiosity, why not just `%notin%`?

Reply:
I just forgot that we have this operator in admiraldev.

Reply:
I see you've updated it. Resolved.